src/Library/Platform/Sdl/CMakeLists.txt

```diff
@@ -32,7 +32,7 @@ target_link_libraries(library_platform_main PRIVATE SDL2::SDL2OE)
 
 add_library(library_platform_sdl STATIC ${PLATFORM_SDL_SOURCES} ${PLATFORM_SDL_HEADERS})
 target_check_style(library_platform_sdl)
-target_link_libraries(library_platform_sdl PUBLIC utility library_logger PRIVATE SDL2::SDL2OE)
+target_link_libraries(library_platform_sdl PUBLIC utility library_logger PRIVATE SDL2::SDL2OE imgui)
```

Comment:
Why are we linking to imgui here?

Reply:
> Why are we linking to imgui here?

Nomore needed since we moved the `ImGui_ImplSDL2_ProcessEvent` elsewhere with the latest changes. I'll remove it.